test/views/wallet/TermsOfUseView/TermsOfUseView.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { TermsOfUseView } from '../../../../app/views/wallet';
+import renderSnapshot from '../../../renderSnapshot';
+
+describe('TermsOfUseView', () => {
```

Comment:
You should test the breadcrumbs until we refactor that logic into a single component.